src/util.rs

```diff
@@ -660,11 +667,17 @@ impl Run {
             .status()
             .unwrap();
 
-        if self.in_test && !self.interp {
+        if self.in_test {
             std::process::Command::new("rm")
-                .arg(executable.clone())
+                .arg(executable.clone() + "-args")
                 .status()
                 .unwrap();
+            if !self.interp {
```

Comment:
why do we only remove the executable when not interpreting?
The tests should always clean up executables.

Reply:
I'm maintaining existing behavior here: https://github.com/egraphs-good/eggcc/blob/main/src/util.rs#L663